implementation/domain/web/jbb-system-web/src/main/java/org/jbb/system/web/session/SessionController.java

```diff
@@ -0,0 +1,98 @@
+ *  You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.jbb.system.web.session;
+
+import org.jbb.system.api.model.session.UserSession;
+import org.jbb.system.api.service.SessionService;
+import org.jbb.system.web.session.data.SessionUITableRow;
+import org.jbb.system.web.session.form.InactiveIntervalTimeForm;
+import org.jbb.system.web.session.form.SessionRemoveForm;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.validation.Valid;
+
+import lombok.extern.slf4j.Slf4j;
+
+
+@Controller
+@Slf4j
+@RequestMapping("/acp/system/sessions")
+public class SessionController {
+
+    private static final String VIEW_NAME = "acp/system/sessions";
+    private static final String USER_SESSION_MODEL_ATTRIBUTE = "userSessions";
+    private static final String SESSION_FORM_NAME = "sessionSettingForm";
+    private static final String SESSION_REMOVE_FORM_NAME = "sessionRemoveForm";
+    private SessionService sessionService;
+
+    @Autowired
+    public SessionController(SessionService sessionService){
+        this.sessionService = sessionService;
+    }
+
+    @RequestMapping(method = RequestMethod.GET)
+    public String getSessionView(Model model){
+
+        InactiveIntervalTimeForm inactiveIntervalTimeForm = new InactiveIntervalTimeForm();
+
+        inactiveIntervalTimeForm.setMaxInactiveIntervalTime(sessionService.getDefaultInactiveSessionInterval().getSeconds());
+        List<UserSession> userSessionList = sessionService.getAllUserSessions();
+
+        List<SessionUITableRow> sessionUITableRowList = userSessionList.stream()
+                .map(userSession ->
+                        new SessionUITableRow(userSession.sessionId(),
+                                        userSession.creationTime(),
+                                        userSession.lastAccessedTime(),
+                                        userSession.usedTime(),
+                                        userSession.inactiveTime(),
+                                        userSession.userName(),
+                                        userSession.displayUserName(),
+                                        userSession.timeToLive()))
+                .collect(Collectors.toList());
+
+        model.addAttribute(USER_SESSION_MODEL_ATTRIBUTE, sessionUITableRowList);
+        model.addAttribute(SESSION_FORM_NAME, inactiveIntervalTimeForm);
+
+        return VIEW_NAME;
+    }
+
+    @RequestMapping(value = "/setnewvalueofproperties",method = RequestMethod.POST)
```

Comment:
is it this `value` not too long? Maybe remove it, then endpoint will be available at `POST /acp/system/sessions`